src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -358,6 +358,7 @@ _outJobInfo(StringInfo str, const Job *node)
 	WRITE_NODE_FIELD(taskList);
 	WRITE_NODE_FIELD(dependedJobList);
 	WRITE_BOOL_FIELD(subqueryPushdown);
+	WRITE_BOOL_FIELD(requiresMasterEvaluation);
```

Comment:
If I happen to merge the 9.6 stuff before you, you'll need to adapt these to the new 9.6 conventions.